Replace missing login strings for translation

aforcier opened the pull request:
I noticed some of our login translations were off. It turns out that this is caused by strings from the login library missing from the base `strings.xml`, either because they were added to the login library without being copied over, or they were removed from the base `strings.xml` because they appeared to be unused.

So this PR adds all missing login strings to the base `strings.xml`, and removes some unused strings from the login library.

@mzorz: It looks like in some cases the dropped strings also have all their translations removed, and sometimes that doesn't seem to happen. In any case I'm not sure if we want to do any manual handling to add back the ones that had translations but were deleted, or if they still exist in Glotpress and they'll just be re-fetched on the next build.

Note: we need a more sustainable fix for this - I was hoping to tackle it for hackweek but it didn't happen. Tracked in #7878, and will look into it soon.

cc @mzorz (also heads up @loremattei)

mzorz commented:
Nice catch @aforcier ! @loremattei here @aforcier is working on a script to make the checks - I'll take care of handling the particular translations in this PR to make them back to Glotpress for 10.2

aforcier commented:
Updated to target `10.2` instead, and also two more small string changes:

1. Updated a String in the login library to reflect an update WordPress-side (no changes to UI or translations at all)
2. Dropped an extra space character from both login and core `strings.xml` that was setting off the diff (was simpler to just remove it since it shouldn't be there anyway)

This can be reviewed and merge whenever @mzorz, I'll work on a script we can add to CI/the release workflow but that can be its own PR.

mzorz commented:
:shipit:

mzorz commented:
Just a note that this has been merged into `release/10.2` and cascaded into `develop` then `master` to give  a few days for translators to fill in the gap 👍
